charts/common/CUSTOM_CONFIG.md.gotmpl

```diff
@@ -0,0 +1,82 @@
+{{- define "custom.custom.configuration.header" -}}
+## Creating a new chart
+{{- end -}}
+
+{{- define "custom.custom.configuration" -}}
+{{ template "custom.custom.configuration.header" . }}
+
+First be sure to checkout the many charts that already use this like [qBittorrent](../qbittorrent/), [node-red](../node-red/) or the many others in this repository.
```

Comment:
Maybe we could also refer to the taskfile / script to create a new chart based on the template chart.